src/Extensions/Services.Remoting/Client/OmexServiceProxyFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.ServiceFabric.Services.Remoting.Client;
+using Microsoft.ServiceFabric.Services.Remoting.V2.FabricTransport.Client;
+
+namespace Microsoft.Omex.Extensions.Services.Remoting.Client
+{
+	/// <summary>
+	/// Class to provide ServiceProxyFactory
+	/// </summary>
+	public static class OmexServiceProxyFactory
+	{
+		/// <summary>
+		/// Instance of the ServiceProxyFactory
+		/// </summary>
+		public static ServiceProxyFactory Instance { get; } = new ServiceProxyFactory(handler =>
```

Comment:
I don't think there is any point of having lazy here since this static field will be initialized only before class usage and in thread save way, so lazy will just add overhead